specifications/api/swagger.yaml

```diff
@@ -281,6 +281,21 @@ x-walletAccountPubkey: &walletAccountPubkey
   maxLength: 128
   example: 1423856bc91c49e928f6f30f4e8d665d53eb4ab6028bd0ac971809d514c92db11423856bc91c49e928f6f30f4e8d665d53eb4ab6028bd0ac971809d514c92db1
 
+x-cosigner: &cosigner
+  type: string
+  pattern: "^(cosigner#)[0-9]+$"
+
+x-cosigners: &cosigners
+  description: Map of cosigners and their account public keys. Use key as in &cosigner, eg. "cosigner#"
+  type: object
+  additionalProperties: *walletAccountPubkey
+  # Note: propertyNames pattern not supported in current OpenAPI version.
+  # propertyNames:
+  #   pattern: '^(cosigner#)[0-9]+$'
+  example:
+    0: { "cosigner#0": "1423856bc91c49e928f6f30f4e8d665d53eb4ab6028bd0ac971809d514c92db11423856bc91c49e928f6f30f4e8d665d53eb4ab6028bd0ac971809d514c92db1"
```

Comment:
I think we need to drop `0:` otherwise it still renders wrong:

![Screenshot from 2021-04-08 11-05-36](https://user-images.githubusercontent.com/42900201/113999567-7148b100-985a-11eb-8517-38d4d705c73a.png)

Reply:
done